04Falcon40bModelFromTweetDatasetBatchCode.ipynb

```diff
@@ -92,7 +92,7 @@
    "source": [
```

Comment:
i think putting all notebooks in a folder `notebooks` should provide more overview for new users of repo.